frontend/public/components/monitoring.tsx

```diff
@@ -641,6 +695,18 @@ const AlertsDetailsPage = withFallback(
         </StatusBox>
       </>
     );
+
+    return activePerspective === 'admin' ? (
+      alertDetails
+    ) : (
+      <NamespacedPage
```

Comment:
It feels wrong to have this here and to be importing components from `@console/dev-console/`.

How about creating a small component in `/dev-console/src/components/monitoring/alerts/MonitoringAlerts.tsx` like

```
<NamespacedPage
  variant={NamespacedPageVariants.light}
  hideApplications
  onNamespaceChange={handleNamespaceChange}
>
  <AlertsDetailsPage
    ...
  />
</NamespacedPage>
```

And export `AlertsDetailsPage` from this file?